src/components/Markdown.react.js

```diff
@@ -11,6 +11,11 @@ import './css/highlight.css';
  * [react-markdown](https://rexxars.github.io/react-markdown/) under the hood.
  */
 class DashMarkdown extends Component {
+    constructor(props) {
+        super(props);
+        this.highlightCode = this.highlightCode.bind(this);
```

Comment:
apparently not needed given our current usage, but makes me nervous to have it there unbound...